### ISSUE TITLE
Think about removing member data compare/correct when eligibility is not checked.

### ISSUE THREAD

**Issue by sclark-rvtd** [RVTD/RideTeam#46]
Right now every time the member eligibility is checked a comparison is done with the state db values and the local db values.  If the data isn't actually received from the state, it probably doesn't need to be checked. 

I'm guessing that members usually only call once a day, so this isn't a big deal.  It would be nice to fix it though.

**Comment by shemsargent-rvtd** [comment]
Agreed. It seems fairly unlikely that member data would change at the State more than once a day. We should skip the modal popup in two cases:  
1. No data from the State (either check skipped because it was already performed today or failure to get data from the State)  
1. Data from State matches data in Member record.
